app/components/post/index.js

```diff
@@ -78,33 +77,36 @@ function makeMapStateToProps() {
                 }
             }
         }
-        const {deviceWidth} = getDimensions(state);
 
         const isAdmin = checkIsAdmin(roles);
         const isSystemAdmin = checkIsSystemAdmin(roles);
 
         let canDelete = false;
         let canEdit = false;
+        let canEditUntil = -1;
         if (post) {
             canDelete = canDeletePost(state, config, license, currentTeamId, currentChannelId, currentUserId, post, isAdmin, isSystemAdmin);
             canEdit = canEditPost(state, config, license, currentTeamId, currentChannelId, currentUserId, post);
+            if (canEdit && license.IsLicensed === 'true' &&
+                (config.AllowEditPost === General.ALLOW_EDIT_POST_TIME_LIMIT || (config.PostEditTimeLimit !== -1 && config.PostEditTimeLimit !== '-1'))
+            ) {
+                canEditUntil = post.create_at + (config.PostEditTimeLimit * 1000) + 1000;
```

Comment:
Why additional `+ 1000`?

Reply:
just following what the original function does https://github.com/mattermost/mattermost-redux/blob/master/src/utils/post_utils.js#L86

Reply:
I can't exactly tell if that's same.  Changes above seems to add 1 sec on top of the `PostEditTimeLimit` set value.  I'd like to think that it should only be:
```
canEditUntil = post.create_at + (config.PostEditTimeLimit * 1000);
```

Reply:
I thought so too, but I wasn’t sure, let me change that